static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx

```diff
@@ -0,0 +1,200 @@
+import {Component, createRef} from 'react';
+import {Theme, withTheme} from '@emotion/react';
+import type {PieSeriesOption} from 'echarts';
```

Comment:
pieChart exists. it doesn't seem to be used anywhere else but just to be careful I created a new component since I ended up modifying it quite a bit